assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/buttons/PriceModifierActions.tsx

```diff
@@ -1,24 +1,27 @@
-import React from 'react';
+import React, { CSSProperties } from 'react';
 
 import AddPriceModifierButton from './AddPriceModifierButtonData';
 import DeletePriceModifierButton from './DeletePriceModifierButton';
-import { TpcModifierFormRowProps } from '../types';
+import { PriceModifierProps } from '../types';
 
-const PriceModifierActions: React.FC<TpcModifierFormRowProps> = ({
-	index,
-	name,
-	price,
-	fields: { push, remove, reset, sort },
-}) => {
+// just temporary
```

Comment:
some day maybe we will be using `emotion` for this kind of scope styles, not today :)